Scripts/script-ExtractDomainFromUrlFormat.yml

```
@@ -10,11 +10,19 @@ script: |-
 
   inp = demisto.args()['input']
 
+  results = []
+
   is_email = validate_email(inp)
   if is_email:
-      domain = re.search("@[\w.]+", inp)
+      domain = re.search("@[\w.|\-]+", inp)
       if len(domain.group()) > 0:
```

this will always be true in case the regex return any result.
if the regex returns without any result in will return None and this command will failed.

```suggestion
      if domain is not None:
```